### ISSUE TITLE
Fix 2.0.1 release on NPM

### ISSUE THREAD

**Issue by 1337LutZ** [tdeNL/tde-webpack-mjml-plugin#16]
Hello,

Im currently upgrading one of our setups, and I noticed that you released a 2.0.1 version. However on NPM it is still defined as 2.0.0: https://www.npmjs.com/package/tde-webpack-mjml-plugin

This gives errors when upgrading to webpack 5 unfortunately :(

Is this something you could look into to fixing?

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for tde-webpack-mjml-plugin@^2.0.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

**Comment by jordikroon** [comment]
Thanks for flagging this @1337LutZ!
A fix (2.0.2) has just been pushed to both NPM and Github that should resolve this issue.

**Comment by 1337LutZ** [comment]
Awesome, thanks for the quick fix. It works 👍🏻
